cleanup-passes/readme.ts

```diff
@@ -145,6 +99,72 @@ ${injectAutodetectedLanguage(behavior.desc)}
   }
 }
 
+function extractDocumentationInfo(element: ElementRepo) {
```

Comment:
It's a private function (called just once, and in this file) and the return type is complicated. My preference would be to let inference handle this one.